conan/tools/cmake/cmake.py

```diff
@@ -104,6 +104,10 @@ def configure(self, variables=None, build_script_folder=None, cli_args=None):
         arg_list.extend(['-D{}="{}"'.format(k, v) for k, v in self._cache_variables.items()])
         arg_list.append('"{}"'.format(cmakelist_folder))
 
+        verbosity = self._verbosity
+        if verbosity is not None:
+            arg_list.append(f"--log-level={verbosity}")
```

Comment:
@RubenRBS I believe unrelated to https://github.com/conan-io/conan/issues/12239. `--log-level` doesn't control what is displayed during build. https://github.com/conan-io/conan/issues/12239 is asking for a conf enabling https://cmake.org/cmake/help/latest/variable/CMAKE_VERBOSE_MAKEFILE.html in CMake, to display all build commands.

Reply:
You're right @SpaceIm in that the conf is mapping to flags with different meanings in each build system, this was somewhat on purpose, but you're right that it's unexpected. Let me rethink this with the team :)